Improved wording in running Django’s test suite in contributing tutorial. (#15335)
django/django: docs/intro/contributing.txt

suite before, it's a good idea to run it once beforehand to get familiar with
its output.

> Before running the test suit, enter django “tests/” directory by running the command 
> “cd tests” and then install its dependencies by running:

Reviewer: We should use double backticks, `Django` should be capitalized, also `the` is missing:
```suggestion
Before running the test suit, enter the Django ``tests/`` directory using the
``cd tests`` command, and install test dependencies by running:
```

Author: ok, I will take care of these. Thank you to guide me
